invisible_cities/reco/xy_algorithms.py

```diff
@@ -58,13 +59,15 @@ def count_masked(cs, d, datasipm, is_masked):
     return np.count_nonzero(~is_masked.astype(bool)[indices])
 
 
-def corona(pos, qs, all_sipms,
-           Qthr            =  0 * units.pes,
-           Qlm             =  5 * units.pes,
-           lm_radius       =  0 * units.mm,
-           new_lm_radius   = 15 * units.mm,
-           msipm           =  3,
-           consider_masked = False):
+def corona( pos             : np.ndarray
```

Comment:
Good catch. I will update it.

Reply:
Wait, the usage example does specify all arguments. Unless you mean the K1, K2, and K3 values, which are left unspecified because the description of the functionality does not depend on those.

Reply:
@adamredwine ?